### PR TITLE
✨ allow birthYear to be null

### PR DESCRIPTION
**Pull request by astroash**
- test User.add works with null value
- update types
- update Cb.getVisitLogAggregates to get null value age group
- add test data for visitor with null age
- update all tests due to new user
- escalate/deescalate routes return `null` for consistency

#289

### CODE REVIEW

**Comment by astroash** [comment]
@eliascodes if possible I would like to merge this before #300 as I would like to use the newly created test user seeds for writing tests on that PR. Creating an extra user for an org throws loads of other tests off which has been fixed in this PR.

File: database/migrations/sql/20180604_06_create_users.sql [hunk 1]
```diff
@@ -44,7 +44,7 @@ CREATE TABLE user_account (
   email                            CITEXT UNIQUE,
   phone_number                     VARCHAR(20),
   post_code                        CITEXT,
-  birth_year                       INT,
+  birth_year                       INT DEFAULT NULL,
```
Comment (eliascodes) [inline database/migrations/sql/20180604_06_create_users.sql]: We've gone live now, we cannot change any of the migrations any more. Any changes must be done with a new migration.

Comment (astroash) [reply database/migrations/sql/20180604_06_create_users.sql]: 😭
